dwellingplace/models/_utils.py

```diff
@@ -83,14 +83,13 @@ def save_xlsx(data, path):
         log.debug("Header: %s", header)
 
         # Add data rows
-        index = 1
-        for datum in data:
-            row = [datum.get(key, None) for key in header]
-            if len(list(filter(None, row))) <= 4:
-                log.debug("Skipping Row since it is mostly Nones: %s", row)
-            else:
-                worksheet.write_row(index, 0, row)
-                index += 1
+        for row, datum in enumerate(data, start=1):
+            # TODO: skip empty rows
```

Comment:
^ that refers to the "skip empty rows"

Regarding the index+=1 instead of enumerate... that is because of the skipping, we need to increment the counter ourselves.